src/include/daos/tse.h

```diff
@@ -38,7 +39,7 @@
 /* 8 bytes used for public members */
 #define TSE_PRIV_SIZE		1016
 /* tse_task arguments max length */
-#define TSE_TASK_ARG_LEN		888
+#define TSE_TASK_ARG_LEN		880
```

Comment:
was there a reason for this?

Reply:
Yes, for the new tse_task_private.dtp_wakeup_time.